src/components/ChapterRouter.js

```diff
@@ -31,6 +31,10 @@ class ChapterRouter extends Component {
                   baseUrl={bookUrl + chapter.url}
                   index={index + 1}
                   question={question}
+                  navigation={{
+                    previous: index > 0,
+                    next: index < chapter.questions.length - 1
+                  }}
                 />
```

Comment:
What do you think about sending `chapter.questions.length` down as props and then making the navigation object in `Question.js`? That way we are only sending down what we absolutely need to.